Formula/kim-api-v2.rb

```diff
@@ -0,0 +1,24 @@
+class KimApiV2 < Formula
+  desc "The Knowledgebase of Interatomic Models (KIM) API"
+  homepage "https://openkim.org"
+  url "https://s3.openkim.org/kim-api/kim-api-v2-2.0.0.txz"
+  sha256 "beb9eb47cfe9ac9a4ad22528da89a67415a5b033075a9318a5d4a800dd4a3f3d"
+  depends_on "doxygen" => :build
```

Comment:
Congratulations @ellio167 for submitting your first PR to the Homebrew project !
There should be an empty line between the sha and the depends section.